lib/modules/tests/test.js

```diff
@@ -0,0 +1,326 @@
+const async = require('async');
+const AccountParser = require('../../utils/accountParser');
+const EmbarkJS = require('embarkjs');
+const utils = require('../../utils/utils');
+
+class Test {
+  constructor(options) {
+    this.options = options || {};
+    this.simOptions = {};
+    this.events = options.events;
+    this.logger = options.logger;
+    this.ipc = options.ipc;
+    this.configObj = options.config;
+    this.ready = true;
+    this.firstRunConfig = true;
+    this.error = false;
+    this.contracts = {};
+    this.firstDeployment = true;
+    this.needConfig = true;
+    this.provider = null;
+    this.accounts = [];
+  }
+
+  init(callback) {
+    this.gasLimit = 6000000;
+    this.events.request('deploy:setGasLimit', this.gasLimit);
+    if (this.options.node !== 'embark') {
+      this.showNodeHttpWarning();
+      return callback();
+    }
+    if (!this.ipc.connected) {
+      this.engine.logger.error("Could not connect to Embark's IPC. Is embark running?");
+      process.exit(1);
+    }
+    return this.connectToIpcNode(callback);
+  }
+
+  connectToIpcNode(cb) {
+    this.ipc.request('blockchain:node', {}, (err, node) => {
+      if (err) {
+        this.logger.error(err.message || err);
+        return cb();
+      }
+      this.options.node = node;
+      this.showNodeHttpWarning();
+      cb();
+    });
+  }
+
+  initWeb3Provider(callback) {
+    if (this.simOptions.accounts) {
+      this.simOptions.accounts = this.simOptions.accounts.map((account) => {
+        if (!account.hexBalance) {
+          account.hexBalance = '0x8AC7230489E80000'; // 10 ether
```

Comment:
I guess, also this hex number would look beautiful in a constant. We could even make it descriptive so that the comment will be obsolete. Something like `BALANCE_10_ETHER_IN_HEX`